notebooks/models/Evaluate flair models.ipynb

```diff
@@ -123,20 +1098,105 @@
     "flair_ner = 'ner'\n",
     "flair_ner_fast = 'ner-fast'\n",
     "flair_ontonotes = 'ner-ontonotes-fast'\n",
-    "flair_bert_embeddings = '../../models/presidio-ner/flair-bert-embeddings.pt'\n",
-    "glove_flair_embeddings = '../../models/presidio-ner/flair-embeddings.pt'\n",
-    "models = [flair_bert_embeddings, glove_flair_embeddings, flair_ner,flair_ner_fast,flair_ontonotes]"
+    "best_model = '../../models/resources/taggers/presidio-ner/best-model.pt'\n",
+    "final_model = '../../models/resources/taggers/presidio-ner/final-model.pt'\n",
```

Comment:
Didn't fully test originally because I didn't have time to run everything. 

It seems the files produced in the first step are named differently from those referred.

Still haven't fully tested as I need to pack up for now. Will confirm it works a bit later.